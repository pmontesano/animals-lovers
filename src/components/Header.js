import logo from '../logo.svg';

const Header = () => {
  return (
    <header className="app-header">
      <img src={logo} className="app-logo" alt="Animal Lovers" />
    </header>
  );
};

export default Header;
