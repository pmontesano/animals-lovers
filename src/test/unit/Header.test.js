import React from 'react';
import { create } from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Header from '../../components/Header';

configure({ adapter: new Adapter() });

describe('Header', () => {
  it('should render correctly', () => {
    const wrapper = create(<Header />);

    expect(wrapper).toMatchSnapshot();
  });

  it('should be has logo', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.find('img')).toHaveLength(1);
  });
});
