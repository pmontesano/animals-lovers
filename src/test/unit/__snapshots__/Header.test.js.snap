// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<header
  className="app-header"
>
  <img
    alt="Animal Lovers"
    className="app-logo"
    src="logo.svg"
  />
</header>
`;
